content/es/_index.html

```diff
@@ -43,12 +43,12 @@ <h2>El desafío de migrar más de 150 microservicios a Kubernetes</h2>
         <button id="desktopShowVideoButton" onclick="kub.showVideo()">Ver vídeo</button>
         <br>
         <br>
-        <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/" button id="desktopKCButton">Asista a la KubeCon en Norte América del 24 al 28 de Octubre 2022</a>
+        <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/" button id="desktopKCButton">Asista a la KubeCon en Europa del 19 al 22 de Marzo 2024</a>
         <br>
         <br>
         <br>
         <br>
-        <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-europe-2023/" button id="desktopKCButton">Asista a la KubeCon en Europa del 17 al 21 de Abril 2023</a>
+        <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america-2024/" button id="desktopKCButton">Asista a la KubeCon en Norte América del 12 al 15 de Noviembre 2024</a>
```

Comment:
```suggestion
        <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america-2024/" button id="desktopKCButton">Asiste a la KubeCon en Norteamérica del 12 al 15 de Noviembre 2024</a>
```

Reply:
Great catch, I have to investigate it on the [RAE](https://www.rae.es/dpd/Norteam%C3%A9rica)

Reply:
Done, thanks